app/javascript/components/Home/Home.tsx

```diff
@@ -1,14 +1,72 @@
 import React from 'react';
-import { Link } from "react-router-dom";
 import { useTranslation } from 'react-i18next';
+import { TFunction } from 'i18next';
+import { CardDeck, Col, Container, Row } from 'react-bootstrap';
+import { HomeCard } from './components';
+
+function getHomeCard(
+    buttonText: string,
+    linkPath: string,
+    text: string,
+    header: string,
+    disabled: boolean,
+): JSX.Element {
+    return (
+        <HomeCard
+            buttonText={buttonText}
+            buttonLinkPath={linkPath}
+            cardText={text}
+            disabled={disabled}
+            header={header}
+        />
+    );
+}
+
+function getOrderGroceriesCard(t: TFunction): JSX.Element {
+    return getHomeCard(
+        t('home.order_groceries'),
+        "/groceries",
+        t('home.order_explanation'),
+        t('home.order_header'),
+        false
+    );
+}
+
+function getViewOrderCard(t: TFunction): JSX.Element {
+    return getHomeCard(
+        t('home.view_order'),
+        "/",
+        t('home.view_order_explanation'),
+        t('home.view_order_header'),
+        true
+    );
+}
+
+function getDeliverCard(t: TFunction): JSX.Element {
+    return getHomeCard(
+        t('home.deliver'),
+        "/",
+        t('home.deliver_explanation'),
+        t('home.deliver_header'),
+        true
+    );
+}
 
 function Home(): JSX.Element {
     const { t } = useTranslation();
 
     return (
-        <Link to="/groceries" role="button">
-            {t("home.make_order")}
-        </Link>
+        <Container>
+            <Row className="justify-content-md-center" style={{ justifyContent: "center", alignItems: "center", height: "80vh" }}>
```

Comment:
These styles will need to be moved to a different class. I added that in our backlog.